airgun/views/host_new.py

```diff
@@ -20,6 +21,14 @@
 from airgun.widgets import SatTableWithoutHeaders
 
 
+class Search(TextInput):
```

Comment:
Search is an already different widget exist in Airgun, I would try to use the different name here. Are able to see this custom behaviour in other places than Host ? If yes, good to put this behaviour in the `airgun.widgets`

Reply:
I was not able to find this behavior in other places. What would you suggest for the class name? :thinking:

Reply:
`SearchInput` how that feels?  I just thought to avoid the same name we have already in Airgun.  I'm pretty open to anything you suggest.